Resume.html

```diff
@@ -0,0 +1,151 @@
+<html>
+
+<head>
+<link rel="stylesheet" type="text/css" href="ResumeCss.css">
+<title>Resume-Team GOLD</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+<div id="First">
+  <div id="Top">
+	<div id="InsideTop">
+		<div id="InsideInsideTop">
+			<p id="first-Paragraph"><center>	TEAM  GOLD</center></p>
+			<p class="italic"><center class="italic"> Software Developer</center></p>
+		</div>
+	</div>
+  </div>
+  <div id="Second-Top">
+	<div id="profile-div">
+	 <p class="Heading"> PROFILE </p>
+	 <p id="profile_data">We are Team gold, Our team members are Mohit, Vishwas, Pallvi. We are great at building web design and provide a better user expereince to end users.
+	  Best practice  to always follow is to work with Agile methodolgy and our team sincerely follows it.
+	  We always distribute our work and work on component level.Later on we merge all our components and try to deliver the product to the best of our capabilities. </p>
+	</div>
+	<div id="pic-div">
+	<img class="profilePic" src="team2.jpg" alt="Vishwas">
+	</div>
+	<div id="address-div">
+	 <p class="Heading">  </p>
+	 <p id="icon"> <img class="icons" src="Home.png"/>Nagla Jula, Karhal Road, Mainpuri,U.P.(205001)</p>
+	 <p id="icon"> <img class="icons" src="mail.png"> vchauhan@tavisca.com</p>
+	 <p id="icon"> <img class="icons" src="company.png"> Tavisca Solutions Pvt. Ltd.</p>
+	 <p id="icon"> <img class="icons" src="Phone4.png"> 9760462946</p>
+	 </div>
+  </div>
+  <div>
+<div id="Div_Education" align="center"> <b>EDUCATION </b></div> <hr align="left "id="line3" size="6px" width="19%" noshade="noshade"> <div id="Div_Skills" align="center"> <b>SKILLS</b> </div> <hr align="left "id="line4" size="5px" width="19%" noshade="noshade"> <div id="Div_Expertise" align="center"> <b>EXPERTISE</b></div>

```

Comment:
keep it in separate lines if its a big content in single line. So it will be readable